website/docs/r/bigip_ltm_virtual_server.html.markdown

```diff
@@ -79,7 +79,7 @@ resource "bigip_ltm_virtual_server" "https" {
 
 * `source` -  (Optional) Specifies an IP address or network from which the virtual server will accept traffic.
 
-* `rules` - (Optional) The iRules you want run on this virtual server. iRules help automate the intercepting, processing, and routing of application traffic.
+* `irules` - (Optional) The iRules you want run on this virtual server. iRules help automate the intercepting, processing, and routing of application traffic.
```

Comment:
:+1: maybe just specify "The iRules list...", just so it'll be clear it's a list.

Reply:
@ncsurfus ?

Reply:
Added @dannyk81 Mentioned change in  https://github.com/terraform-providers/terraform-provider-bigip/pull/172 and merged